docs/working-with-projects/working-with-images/managing-helm-charts.md

```diff
@@ -74,9 +76,9 @@ Clicking the chart version number link will open the chart version details view.
 
 Clicking the `DOWNLOAD` button on the top right will start the downloading process.
 
-## Working with the Helm CLI
+## Working with ChartMuseum via the Helm CLI
 
-As a helm chart repository, Harbor can work smoothly with Helm CLI. About how to install Helm CLI, please refer [install helm](https://docs.helm.sh/using_helm/#installing-helm). Run command `helm version` to make sure the version of Helm CLI is v2.9.1+.
+As a helm chart repository, Harbor can work smoothly with Helm CLI. About how to install Helm CLI, please refer [install helm](https://helm.sh/docs/intro/install/). Run command `helm version` to make sure the version of Helm CLI is v2.9.1+.
```

Comment:
-> 'As a helm chart repository, Harbor can work interoperate with Helm CLI. To install Helm CLI, please refer to [install helm](https://helm.sh/docs/intro/install/). Please also run `helm version` command to make sure the version of Helm CLI is v2.9.1+.'